src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -383,7 +383,7 @@ export default class DefaultDeviceController
 
       await this.chooseAudioTransformInputDevice(device);
     } else {
-      this.logger.info(`Choosing intrinsic input device ${device}`);
+      this.logger.info(`Choosing intrinsic audio input device ${JSON.stringify(device)}`);
```

Comment:
Don't think we should JSON.stringify device here. It should be a string object :)

Reply:
![image](https://user-images.githubusercontent.com/54328451/120551916-71bd7e00-c3ab-11eb-9b96-cc2af7799828.png)

I can pass in a MedaStreamConstraint also - in which case I'll see this

Reply:
Reverted.